app/Filament/Widgets/StatsOverview.php

```diff
@@ -5,19 +5,40 @@
 use App\Models\Response;
 use Filament\Widgets\StatsOverviewWidget as BaseWidget;
 use Filament\Widgets\StatsOverviewWidget\Stat;
+use Flowframe\Trend\Trend;
+use Flowframe\Trend\TrendValue;
 
 class StatsOverview extends BaseWidget
 {
     protected function getStats(): array
     {
         $responsesTotal = Response::query()->count();
         $responses7d = Response::query()->since(now()->subWeek())->count();
+        $responses7dChartData =Trend::model(Response::class)
```

Comment:
It seems like the auto formatter did not run. Try `make format` to run it manually.